packages/calypso-e2e/src/lib/components/sidebar-component.ts

```diff
@@ -35,9 +35,12 @@ export class SidebarComponent {
 	 * Waits for the WordPress.com Calypso sidebar to be ready on the page.
 	 */
 	async waitForSidebarInitialization(): Promise< void > {
-		await this.page.waitForLoadState( 'load' );
 		const sidebarLocator = this.page.locator( selectors.sidebar );
-		await sidebarLocator.waitFor();
+
+		await Promise.all( [
+			this.page.waitForLoadState( 'load', { timeout: 20 * 1000 } ),
+			sidebarLocator.waitFor( { timeout: 20 * 1000 } ),
+		] );
```

Comment:
@karenroldan This should take care of the SidebarComponent timeout you mentioned in https://github.com/Automattic/wp-calypso/pull/70507.